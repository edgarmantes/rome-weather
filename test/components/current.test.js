import React from 'react'
import renderer from 'react-test-renderer';

import PropTypes from 'prop-types';
import CurrentContainer, { Current } from '../../src/components/current';

describe('<Current />' , () => {


	let wrapper

	beforeEach( ()=>{
		wrapper = shallow(<Current unit={"C"} />)
	})

	it('Renders Current component properly', () => {
		
		// expect(wrapper.length).toEqual(1)
	});

	// it('Adds Day passed in value to this.props.day', () => {
	// 	expect(wrapper.find(".five-day").text()).toEqual("Friday")
	// })

	// it('Adds Date passed in value to this.props.hi', () => {
	// 	let temp = "hi: 80°"
	// 	expect(wrapper.find(".five-hi").text()).toEqual(temp)
	// })

	// it('Adds Date passed in value to this.props.lo', () => {
	// 	let temp = "lo: 60°"
	// 	expect(wrapper.find(".five-lo").text()).toEqual(temp)
	// })

	// it('Snapshot of DayCast', () => {
 //        const renderedValue =  renderer.create(<Current />).toJSON()
 //        expect(renderedValue).toMatchSnapshot();
 //    });
});
