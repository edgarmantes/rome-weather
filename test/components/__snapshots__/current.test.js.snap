// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Current /> Snapshot of DayCast 1`] = `
<div
  className="current-container"
>
  <div
    className="current-weather"
  >
    <div
      className="current-temp"
    >
      Wait...
      <span
        className="current-symb"
      >
        C
      </span>
    </div>
    <div
      className="current-range"
    >
      <span
        className="current-hi"
      >
        hi: 
        °/
      </span>
      <span>
        lo: 
        °
      </span>
    </div>
    <div
      className="current-humidity"
    >
      hum: 
      %
    </div>
  </div>
  <select
    onChange={[Function]}
    value="C"
  >
    <option
      value="F"
    >
      F°
    </option>
    <option
      value="C"
    >
      C°
    </option>
    <option
      value="Both"
    >
      F°/C°
    </option>
  </select>
  <div
    className="pollut-container"
  >
    <h3>
      Carbon Monoxide at sea level:
    </h3>
    <p>
      ppm
    </p>
  </div>
</div>
`;
